src/pages/Courses.tsx

```
@@ -119,87 +207,238 @@ export default function Courses() {
-                  <TableCell colSpan={6} className="text-center text-muted-foreground">暂无数据</TableCell>
-                </TableRow>
-              )}
-            </TableBody>
-          </Table>
+
+                      try {
+                        await deleteCourse(c.id);
+                        // 删除成功，刷新课程列表
+                        const updatedCourses = await fetchCourses();
+                        setCourses(updatedCourses);
+                      } catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)